torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8780,6 +8780,15 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
         supports_autograd=True,
         supports_forward_ad=True,
     ),
+
+    ForeachFuncInfo(
+        'sign',
+        dtypes=floating_types_and(torch.bool, torch.bfloat16, torch.half),
+        dtypesIfCUDA=floating_types_and(torch.bfloat16, torch.float16),
+        sample_inputs_func=foreach_inputs_sample_func(1, False, False),
```

Comment:
I'm looking in foreach_inputs_sample_func--do you know if we test inplace ops here?

Reply:
It should be covered by `test_unary_op`

Reply:
Ah yes, I see there's _inplace_unary_test now, thank you for the pointer!